schema/core/reusable/atomic_orbital.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$id": "core/reusable/atomic_orbital.json",
```

Comment:
We don't use the file extension in the id. It's usually best to just run the following script to automatically generate the schema Id:
```bash
cd src/js/scripts
node setSchemaIds.mjs
```

Reply:
Thank you Alex, fixed it.